homeassistant/components/sensor/yr.py

```diff
@@ -61,7 +61,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     coordinates = dict(lat=latitude,
                        lon=longitude,
-                       msl=elevation)
+                       msl=int(elevation))
```

Comment:
It should be coerced to `int` then to remove the decimal places added going to a string in the url